russian/kluch-na-14/kluch.md

```diff
@@ -0,0 +1,3 @@
+Неопрятный мужичок с жилеткой поверх пуховика подбежал откуда то сзади и застрял между двух деревьев, пытаясь пропихнуть свой несоизмеримо круглый живот между двух березок. В какой-то момент он понял, что самому ему не выбраться и посмотрел на вас взглядом хищного кота, попавшего в крысиную засаду. Как будто это вы были в этом виноваты. Сразу стало ясно - он русский. Даже в этом взгляде можно было прочитать мольбу о помощи. Ваши действия:
+[С разбегу зарядить в пузень](pinok/pinok.md)
+[Подойти и погладить](pogladit/pogladit.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.